components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt

```diff
@@ -213,7 +218,8 @@ class PromptFeature private constructor(
         onManageLogins = onManageLogins,
         creditCardPickerView = creditCardPickerView,
         onManageCreditCards = onManageCreditCards,
-        onSelectCreditCard = onSelectCreditCard
+        onSelectCreditCard = onSelectCreditCard,
+        addressDelegate = addressDelegate
```

Comment:
Can you file an issue to convert credit cards and logins to also use a similar delegate to clean up the PromptFeature? Also, feel free to take on the issue.

Reply:
Filed https://github.com/mozilla-mobile/android-components/issues/12257 for this.